esp/templates/program/modules/finaidapprovemodule/finaid.html

```diff
@@ -0,0 +1,71 @@
+{% extends "main.html" %}
+
+{% block title %}Approve Financial Aid Requests for {{program.niceName}}{% endblock %}
+
+{% block stylesheets %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="/media/styles/forms.css" />
+    <style>
+    #program_form {
+      overflow: auto;
+      max-height: 500px;
+      width: 700px;
+      margin: 0px 10px 15px 10px;
+    }
+    </style>
+{% endblock %}
+
+{% block xtrajs %}
+<script type="text/javascript" language="javascript">
+function toggle(source) {
+  checkboxes = document.getElementsByName('user');
+  for(var i=0, n=checkboxes.length;i<n;i++) {
+    checkboxes[i].checked = source.checked;
+  }
+}
+</script>
+{% endblock %}
+
+{% block content %}
+<div style="padding-bottom: 1.1em;">
+  <h1>Financial Aid Requests for {{ program.niceName }}</h1>
+  <p>Below is a list of users that have requested financial aid.</p>
+  <p>Please select the requests that you would like to approve.</p>
+
+  <form action="/manage/{{ program.getUrlBase }}/finaidapprove/" method="post">
+    <div id="program_form">
+      <table align="center">
+        <tr>
+          <th><input type="checkbox" onClick="toggle(this)" /></th>
+          <th>Approved?</th>
+          <th width="70px">Name</th>
+          <th width="70px">Household Income</th>
```

Comment:
Can you also include the `reduced_lunch` and `student_prepare` attributes as well?

Reply:
Added! What should I label the student_prepare column in the template?

Reply:
Well, the help text for this field is "Did anyone besides the student fill out any portions of this form?" (True then means that someone besides the student prepared the answers), which seems to be the opposite of the field name (which implies that True means that the student prepared the answers). I was thinking something like "Student Prepared" or "Prepared by Student" would be a good label for the column, but then you'd want to show the inverse of the actual field value (I think). If you don't want that, you could use "Non-Student Prepared" or "Prepared by non-Student" or something like that, I guess?

Reply:
I went with "Filled Out Only by Student". Does that sound good? I don't want some unsuspecting admin to be confused by what it's trying to portray. :)